### ISSUE TITLE
Add approval feature to users/ members for client admin

### ISSUE THREAD

**Issue by innocentmm** [codersforcauses/home2health#101]
Only if not already taken care of by auth.....

**Comment by frinzekt** [comment]
Hey @innocentmm , I just want to clarify what this actually means.

Is it approval from an admin user role to become a member?

**Comment by innocentmm** [comment]
Yes so, before the client had requested a blog site accessible to normal users and a separate members area. We were able to convince them to set the users specifically as members to reduce duplication.

So now, with the user registration, we need to ensure that there’s is a feature that forces the user registration process to only be completed (i.e user creation) once an admin member approves the registration.

Kind regards,

Innocent.
________________________________

**Comment by frinzekt** [comment]
Okay, so to clarify a few things:
- Normal users as users that are not registered and they are able to see posts, but cannot create post
- Members are users that are FULLY REGISTERED (with approval from an admin) and can create a post. (currently what we have is that a user could register and become a member [without admin approval]. From there they could create post.)

Are there any other rules? like are there any exclusive content from posts that only members could see, but not for normal users?

**Comment by JKKim98** [comment]
@frinzekt Let's keep things simple and @innocentmm let's not give the clients any ideas that increases complexity :P

**Comment by innocentmm** [comment]
No that’s it.

Kind regards,

Innocent.
________________________________

**Comment by innocentmm** [comment]
It’s not complicating it, it simplifies things by reducing the need for an additional members area

Kind regards,

Innocent.
________________________________
